oauth2/jwt.go

```diff
@@ -105,13 +107,14 @@ func (j *JWT) Create(ctx context.Context, user Principal) (Token, error) {
 	// Create a new token object, specifying signing method and the claims
 	// you would like it to contain.
 	claims := &Claims{
-		gojwt.StandardClaims{
+		StandardClaims: gojwt.StandardClaims{
 			Subject:   user.Subject,
 			Issuer:    user.Issuer,
 			ExpiresAt: user.ExpiresAt.Unix(),
 			IssuedAt:  user.IssuedAt.Unix(),
 			NotBefore: user.IssuedAt.Unix(),
 		},
+		Organization: user.Organization,
```

Comment:
You could add scheme here (this is in the oauth2 package so, scheme should be hardcoded to "oauth2" here)

Reply:
I think I'd like to put this in a separate PR, unless you think it would be better to add it here. Opened https://github.com/influxdata/chronograf/issues/2201